### ISSUE TITLE
Translate validation error messages

### ISSUE THREAD

**Issue by nichtich** [pkiraly/metadata-qa-marc-web#90]
Validation error messages are created in plain English, e.g. "there are *x* instances". These should also be made translatable, e.g with message string "there are %d instances" (en) = "kommt %d mal vor" (de).

**Comment by pkiraly** [comment]
It is a nice workaround, but the proper solution would be done inside the analysis, and separate the raw data, and message template there. Not only in this case, but in every other error types. It would require more work. So I reopen this issue, move it to another milestone, and create another ticker in the backed.
